OSDOCS-12321-RN#Adding WIF storage support (#89135)
openshift/openshift-docs: storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc

* The _GCP Filestore CSI driver_ enables you to create and mount GCP Filestore PVs.

> {product-title} v4.17.11 {gcp-short} Filestore supports Workload Identity. This allows users to access Google Cloud resources using federated identities instead of a service account key. {gcp-wid-short} must be enabled globally during installation, and then configured for the {gcp-short} Filestore CSI Driver Operator. For more information, see xref:../../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-olm-operator-install_persistent-storage-csi-google-cloud-file[Installing the {gcp-short} Filestore CSI Driver Operator].

Reviewer: ```suggestion
{product-title} v4.17.11 {gcp-short} Filestore supports Workload Identity. You can now access Google Cloud resources by using federated identities instead of a service account key. 

You must enable {gcp-wid-short} globally during installation, and then configure for the {gcp-short} Filestore CSI Driver Operator. For more information, see xref:../../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-olm-operator-install_persistent-storage-csi-google-cloud-file[Installing the {gcp-short} Filestore CSI Driver Operator].
```

- Same comment: Avoid using "allow": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a
- Active voice vs passive voice
- You might want to break this para just like you did in the release notes module.
